src/test/java/com/yelp/nrtsearch/server/plugins/PluginsServiceTest.java

```diff
@@ -122,22 +127,28 @@ public void testGetPluginJars() throws IOException {
     folder.newFile("not_jar");
     folder.newFile("some_file.txt");
     folder.newFolder("some_folder.jar");
-    PluginsService pluginsService = new PluginsService(getEmptyConfig());
+    PluginsService pluginsService = new PluginsService(getEmptyConfig(), getCollectorRegistry());
     assertEquals(
         new HashSet<>(jars), new HashSet<>(pluginsService.getPluginJars(folder.getRoot())));
   }
 
-  public static class LoadTestPlugin extends Plugin {
+  public static class LoadTestPlugin extends Plugin implements MetricsPlugin {
```

Comment:
Maybe instead making this class a `MetricsPlugin`, create a second class and add a separate test to verify it gets the registry and this one doesn't.

Reply:
Done.